backend/nunjucksSetup.test.js

```diff
@@ -8,19 +8,18 @@ describe('toUserSearchFilter', () => {
   it('should show filter headings', () => {
     const result = njk.getFilter('toUserSearchFilter')({}, [], [], '', true)
     expect(result.heading.text).toBe('Filter')
-    expect(result.selectedFilters.heading.text).toBe('Selected filters')
     expect(result.selectedFilters.clearLink.text).toBe('Clear filters')
   })
 
   describe('user filter', () => {
     it('should show current user filter section', () => {
```

Comment:
all these test now change to show there is no section when no filter for that category is selected